cookbooks/bcpc-hadoop/attributes/maintenance.rb

```diff
@@ -0,0 +1,10 @@
+default[:bcpc][:hadoop][:acl_group] = "acl_group_for_pam"
+default[:bcpc][:hadoop][:base_dn] = "DC=domain,DC=example,DC=com"
```

Comment:
Since we use `BCPC.EXAMPLE.COM` as our domain in Test-Cluster, shouldn't this be `"DC=bcpc,DC=example,DC=com"`?